style.css

```diff
@@ -0,0 +1,30 @@
+html {
```

Comment:
We should probably not style the `<html>` maybe use other containers in the body